src/program/lwaftr/selftest.sh

```diff
@@ -17,8 +17,13 @@ echo "Testing snabb lwaftr bench"
     ${TDIR}/tcp-frominet-bound.pcap ${TDIR}/tcp-fromb4-ipv6.pcap
 
 echo "Testing snabb lwaftr bench --reconfigurable"
-./snabb lwaftr bench --reconfigurable -D 0.1 ${TDIR}/icmp_on_fail.conf \
-    ${TDIR}/tcp-frominet-bound.pcap ${TDIR}/tcp-fromb4-ipv6.pcap
+./snabb lwaftr bench --reconfigurable -D 1 ${TDIR}/icmp_on_fail.conf \
+    ${TDIR}/tcp-frominet-bound.pcap ${TDIR}/tcp-fromb4-ipv6.pcap &
+
+LEADER_PID=$!
```

Comment:
Plausibly. I originally had a sleep there, then removed it, because it wasn't appearing to be a problem in practice (and sleep is always technically the wrong solution.)